os/test/src/PathTests.scala

```diff
@@ -6,12 +6,15 @@ import os._
 import utest.{assert => _, _}
 import java.net.URI
 object PathTests extends TestSuite {
+  val wintest = Option(System.getenv("WINPATHTEST")).nonEmpty
```

Comment:
I added this as a convenience for experimentally enabling all tests in Windows, to see which ones fail and why.  Ideally, the platform-conditional tests can be eliminated or minimized.

Reply:
Some of the tests are currently disabled because expressions that are equal on other platforms differ in `Windows` by the presence or absence of a drive letter.  For example:
```scala
(root / "omg").wrapped == "C:\\omg"
Paths.get("/omg")      == "\\omg"
```
Other tests, as currently written,  are disabled on`Windows` because of mismatch between forward slashes and backslashes.
The following test would fail because "\\src\\main\\scala"  != "/src/main/scala".
```scala
val d = pwd
val abs = d / base
test("Constructor") {
  assert(
    abs.toString.drop(d.toString.length).norm == "/src/main/scala",
    abs.toString.length > d.toString.length
  )
}
```
I will revert `os/test/src/PathTests.scala` and submit a separate PR, since it's a separate concern.